Resources/Prototypes/_DV/Actions/types.yml

```diff
@@ -48,3 +48,16 @@
     checkCanInteract: false
     checkConsciousness: false
     event: !type:PrecognitionPowerActionEvent
+
+- type: entity
+  id: ActionChitzite
+  name: action-name-chitzite
+  description: action-description-chitzite
```

Comment:
doesnt need to use locale, downstreams can translate with entity locale overrides